resources/keywords/import_provider_account.resource

```diff
@@ -13,6 +13,11 @@ ${provaccname}        ${EMPTY}
 ${INV_ORG_ID}         invalid_org_id
 ${INV_PUB_KEY}        invalid_pub_key
 ${INV_PRI_KEY}        invalid_private_key
+${INV_AWS_ACCS_KEY_ID}  invalid_aws_access_key_id
```

Comment:
These variables also can be removed from line 16 to 18

Reply:
Please look at the previous comment.